data/mods/Magiclysm/items/tools.json

```diff
@@ -0,0 +1,23 @@
+[
+  {
+    "id": "cauldron_demon_chitin",
+    "type": "CONTAINER",
+    "category": "other",
+    "name": { "str": "cauldron of purification", "str_pl": "cauldrons of purification" },
+    "description": "This cauldron made of demon spider chitin seems to absorb the light.  It will hold 16 liters of material and will absorb poisons from it.  It may have other properties that require discovery.",
+    "weight": "1424 g",
```

Comment:
sorry i didn't catch this earlier, but for a 20 L object, 3 pounds is very very light.

Reply:
I agree, but I used straight math for demon chitin and bowl volume.  If the cauldron contains 16 L of volume that means 4 L of material.  Each chunk of demon chitin is 250 ml so 16 chunks to make the cauldron.  4 L of Demon chitin chunks at 89 grams per 250 ml comes out to 1424 exactly.  I'm good to update it to whatever you think is best but I feel it's important to show my math and why I will need you to let me know what it should actually be?  I can add the bone glue in weight but that only adds 450 g in weight.